<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPICROQU5UCNHO3" ignored="0" name="Test" tests="26" failures="4" timestamp="05 一月 2019 09:04:28 GMT" time="8.45" errors="0">
  <testcase name="f" time="0.416" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.445" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.082" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.105" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="1.521" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.419" classname="com.testapi.casev.Testapi">
    <failure type="java.lang.AssertionError" message="expected [4001] but found [4000]">
      <![CDATA[java.lang.AssertionError: expected [4001] but found [4000]
at com.testapi.casev.Testapi.f(Testapi.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.039" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.045" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.058" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.3" classname="com.testapi.casev.Testapi">
    <failure type="java.lang.AssertionError" message="expected [4001] but found [4000]">
      <![CDATA[java.lang.AssertionError: expected [4001] but found [4000]
at com.testapi.casev.Testapi.f(Testapi.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.03" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.034" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.385" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="1.066" classname="com.testapi.casev.Testapi">
    <failure type="java.lang.AssertionError" message="expected [4001] but found [4000]">
      <![CDATA[java.lang.AssertionError: expected [4001] but found [4000]
at com.testapi.casev.Testapi.f(Testapi.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.05" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.034" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.021" classname="com.testapi.casev.Testapi">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.018" classname="com.testapi.casev.Testapi">
    <failure type="java.lang.AssertionError" message="expected [4001] but found [4000]">
      <![CDATA[java.lang.AssertionError: expected [4001] but found [4000]
at com.testapi.casev.Testapi.f(Testapi.java:41)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.015" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.398" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.065" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.095" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.783" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.506" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.084" classname="com.testapi.casev.Testapi"/>
  <testcase name="f" time="0.032" classname="com.testapi.casev.Testapi"/>
</testsuite> <!-- Test -->
